requirements.txt

```diff
@@ -3,6 +3,7 @@ numpy==1.19.5
 tensorflow==2.5.0
 Werkzeug==2.0.1
 Flask==2.0.1
+PyInstaller==4.5.1
```

Comment:
I think this one should come under dev section, I would call it an optional change as you're anyway gonna update the PR

Reply:
Okay, makes sense